package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
+    "postinstall": "node scripts/postinstall.js",
```

Comment:
This is causing issue for npm install process, did you forget to include this postinstall.js scrypt? @kkkk666